packages/form/src/components/RolePicker/RolePicker.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { ErrorMessage } from "../ErrorMessage";
+import { Controller, useFormContext } from "react-hook-form";
+import { RolePickerBasic } from "./RolePickerBasic";
+
+interface Role {
+  name: string;
+  id: number;
+}
+
+interface IProperties {
+  name: string;
+  label?: string;
+  placeholder?: string;
+  options: Role[];
+}
+
+export const RolePicker = ({
+  name,
+  placeholder,
+  options,
+  label,
+}: IProperties) => {
+  const { control, getFieldState } = useFormContext();
+
+  const { error } = getFieldState(name);
+
+  return (
+    <>
+      <Controller
+        name="city"
```

Comment:
```suggestion
        name={name}
```
Use name from props here